docs/fundamentals/monorepo.md

```diff
@@ -0,0 +1,296 @@
+# Building like Google: Monorepos and Automation
+
+In this guide you will look at one of the most interesting parts of Nx. The part that can make so many things so much easier, that it can have a transformative effect on a team and even on an organization.
+
+What is it?
+
+It is a "monorepo" way of building applications.
+
+Working with multiple applications and libraries is difficult. From Google to Facebook, Uber, Twitter and more, a good amount of large software companies handle this challenge by taking a monorepo approach. And they have been doing so for years. These are some of the advantages this approach provides:
```

Comment:
minor: I feel like a reader might not understand the difference between an application and a library yet.